apps/admin/ithc/01/kustomization.yaml

```diff
@@ -7,5 +7,3 @@ resources:
 
 patchesStrategicMerge:
   - ../../traefik2/ithc/01.yaml
-  - ../../traefik2/ithc/secret-provider.yaml
```

Comment:
This also needs removed from 01
https://github.com/hmcts/cnp-flux-config/blob/clean-01-deploy-to-base/apps/admin/ithc/01/kustomization.yaml#L5

Reply:
& L6

Reply:
& could you remove any traefikv1 files that aren't needed now for ithc?